interactive_engine/executor/ir/graph_proxy/src/adapters/gs_store/read_graph.rs

```diff
@@ -55,19 +56,20 @@ where
 {
     store: Arc<dyn GlobalGraphQuery<V = V, E = E, VI = VI, EI = EI>>,
     partition_manager: Arc<dyn GraphPartitionManager>,
+    is_groot: bool,
```

Comment:
I think `is_groot` may not be good, since we adapt to `GlobalGraphQuery`, regardless of the storage implementations (e.g., we may have more storages support push_down in the future). A choice may be we can use two flags to indicate if `GlobalGraphQuery` supports `row_push_down` or `column_push_down`. By this way, we can also remove the `columns_filter_pushdown` feature.

Reply:
add 2 flags in `GraphScopeStore`:
```rust
pub struct GraphScopeStore<V, VI, E, EI>
where
    V: StoreVertex + 'static,
    VI: Iterator<Item = V> + Send + 'static,
    E: StoreEdge + 'static,
    EI: Iterator<Item = E> + Send + 'static,
{
    store: Arc<dyn GlobalGraphQuery<V = V, E = E, VI = VI, EI = EI>>,
    partition_manager: Arc<dyn GraphPartitionManager>,
    row_filter_pushdown: bool,
    column_filter_pushdown: bool,
}
```